tests/api_test.py

```diff
@@ -41,13 +41,29 @@
 from jax import test_util as jtu
 from jax import tree_util
 from jax import linear_util as lu
+from jax.lib import version
 
 from jax.config import config
 config.parse_flags_with_absl()
 FLAGS = config.FLAGS
 
 
-class JitTest(jtu.JaxTestCase):
+class CPPJitTest(jtu.JaxTestCase):
+  """Shared tests between the Python and the C++ jax,jit implementations.
+
+  Because the Python implementation supports more features, we need to have the
+  Python tests that extend the C++ tests (and not the other way around).
+  """
+
+  @staticmethod
+  def get_jit():
```

Comment:
Couldn't we just have a `self.jit` attribute, and two subclasses that populate `jit` differently?

`self.get_jit()` is a bit wordy.